### ISSUE TITLE
Offload traffic to Packet Broker

### ISSUE THREAD

**Issue by johanstokking** [TheThingsNetwork/lorawan-stack#671]
#### Summary

Offload traffic to Packet Broker.

#### Why do we need this?

For making traffic available to other networks, the Gateway Server should offload traffic to the Packet Broker.

#### What is already there? What do you see now?

All traffic is sent to the cluster's Network Server.

#### What is missing? What do you want to see?

Based on the forwarding table, GS should (also) forward the traffic to the configured Packet Broker(s).

#### How do you propose to implement this?

- Add forwarding table in GS configuration with host and a slice of DevAddr prefixes
- Based on DevAddr of incoming message, see if there's a matching entry in the table
- If so, forward the message there
- The local NS acts in a similar way. This way, we can also reduce DevAddr prefixes for the cluster NS, which is handy for development

#### Can you do this yourself and submit a Pull Request?

Yes

**Comment by johanstokking** [comment]
Blocked by https://github.com/TheThingsIndustries/lorawan-stack/issues/1664

**Comment by johanstokking** [comment]
This is already done
